### PR TITLE
Fix bug in CSRF cookie extraction

### PR DESCRIPTION
**Pull request by adamchainz**
Fix a name bug from #25 / d3131d4.

### CODE REVIEW

File: nexus/media/js/nexus.js [hunk 1]
```diff
@@ -34,7 +34,7 @@ jQuery.ajaxSetup({
         }
 
         if (!safeMethod(settings.type) && sameOrigin(settings.url)) {
-            var cookieName = $('#nexus-constants').data('csrfCookieName');
+            var cookieName = $('#nexus-constants').attr('data-csrf-cookie-name');
```
Comment (graingert) [inline nexus/media/js/nexus.js:37]: you should be able to use the .data call from jQuery as before

Comment (adamchainz) [inline nexus/media/js/nexus.js:37]: Not with the current version of jQuery, 1.4.4

Comment (graingert) [inline nexus/media/js/nexus.js:37]: oh that's pretty old

Comment (john-sandall) [inline nexus/media/js/nexus.js:37]: Tom you don't work at YPlan any more!

Comment (adamchainz) [inline nexus/media/js/nexus.js:37]: Neither do you John! But hey, this is open source.

**Pull request merged**